src/js/components/CandidateListRoot/CandidateListRoot.jsx

```diff
@@ -346,18 +346,18 @@ class CandidateListRoot extends Component {
             </WhatIsHappeningTitle>
           )}
           <MobileArrowsInnerWrapper className="u-show-mobile">
-            <LeftArrowInnerWrapper onClick={() => { handleHorizontalScroll(this.scrollElement.current, HORIZONTAL_SCROLL_SPEED, HORIZONTAL_SCROLL_DISTANCE_MOBILE_ARROW_CLICK, HORIZONTAL_SCROLL_STEP_LEFT, this.checkScrollPositionLocal); }}>
+            <LeftArrowInnerWrapper id="leftArrowMobile" onClick={() => { handleHorizontalScroll(this.scrollElement.current, HORIZONTAL_SCROLL_SPEED, HORIZONTAL_SCROLL_DISTANCE_MOBILE_ARROW_CLICK, HORIZONTAL_SCROLL_STEP_LEFT, this.checkScrollPositionLocal); }}>
```

Comment:
I would recommend naming this something like "candidateLeftArrowMobile" because we also have this set of arrows in WebApp/src/js/components/RepresentativeListRoot/RepresentativeListRoot.jsx